src/content/en/updates/2018/09/inside-browser-part1.md

```diff
@@ -0,0 +1,298 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Learn how browser turn your code into functional website from high-level architecture to the specifics of the rendering pipeline.
+
+{# wf_published_on: 2018-09-05 #}
+{# wf_updated_on: 2018-09-05 #}
+{# wf_featured_image: /web/updates/images/inside-browser/cover.png #}
+{# wf_featured_snippet: Learn how browser turn your code into functional website from high-level architecture to the specifics of the rendering pipeline. In part 1, we’ll take a look at core computing terminology and Chrome’s multi-process architecture. #}
+{# wf_blink_components: N/A #}
+
+<style>
+  figcaption {
+    font-size:0.9em;
+  }
+</style>
+
+# Inside look at modern web browser (part 1) {: .page-title }
+
+## CPU, GPU, Memory, and multi-process architecture
+
+{% include "web/_shared/contributors/kosamari.html" %}
+
+In this 4-part blog series, we’ll look inside the Chrome browser from high-level architecture to 
+the specifics of the rendering pipeline. If you ever wondered how the browser turns your code into 
+a functional website, or you are unsure why a specific technique is suggested for performance 
+improvements, this series is for you. 
+
+As part 1 of this series, we’ll take a look at core computing terminology and Chrome’s 
+multi-process architecture. 
+
+Note: If you are familiar with the idea of CPU/GPU and process/thread you may skip to 
+[Browser Architecture](#browser-architectur).
+
+## At the core of the computer are the CPU and GPU
+
+In order to understand the environment that the browser is running, we need to understand a few 
+computer parts and what they do.
+
+### CPU
+
+<figure class="attempt-right">
+  <img src="../../images/inside-browser/CPU.png" alt="CPU">
```

Comment:
Please use absolute paths, if we move to web.dev or some other place, using absolute URLs will make life easier. (and throughout)